doc/sphinx/practical-tools/utilities.rst

```diff
@@ -92,17 +92,55 @@ CoqMakefile
   is a makefile for ``GNU Make`` with targets to build the project
   (e.g. generate .vo or .html files from .v or compile .ml* files)
   and install it in the ``user-contrib`` directory where the |Coq|
-  library is installed. Run ``make`` with the ``-f CoqMakefile``
-  option to use ``CoqMakefile``.
+  library is installed.
 
 CoqMakefile.conf
   contains make variables assignments that reflect
   the contents of the ``_CoqProject`` file as well as the path relevant to
   |Coq|.
 
+The recommended approach is to embed ``CoqMakefile`` into a standard
```

Comment:
```suggestion
The recommended approach is to invoke ``CoqMakefile`` from a standard
```

Reply:
Also, I want to keep the sentence that says that including the generated makefile is discouraged.